src/components/cardDetail/Comment.tsx

```diff
@@ -0,0 +1,20 @@
+const Comment = () => {
+  return (
+    <div className='flex gap-8'>
+      <div className='h-26 w-26 rounded-full bg-amber-300 text-center' />
+      <div className='flex flex-col'>
+        <div className='flex items-center gap-8'>
+          <h3 className='text-[12px] font-semibold text-gray-700 tablet:text-md/24'>name</h3>
```

Comment:
```suggestion
          <h3 className='text-xs font-semibold text-gray-700 tablet:text-md'>name</h3>
```
text도 크게 차이가 없다면 선언된 속성을 쓰는게 좋을 거 같습니다.(px rem차이가 있기도 하고) text-md에 이미 lineheight가 24로 지정되어있어 이 부분은 빼셔도 될 거 같아요!

Reply:
xs 가 있군요! 넵 수정하겠습니다!